CMakeLists.txt

```diff
@@ -33,6 +33,7 @@ find_package(k4FWCore REQUIRED)
 find_package(Gaudi REQUIRED)
 find_package(DD4hep REQUIRED)
 find_package(k4SimGeant4 REQUIRED)
+find_package(k4geo REQUIRED)
```

Comment:
Is this necessary for everything or just for the test (environment)?

Reply:
Yeah only for testing, so I moved it to the corresponding `CMakeLists.txt`. I should actually add a test